plan/logical_plan_builder.go

```diff
@@ -1134,6 +1150,8 @@ func (b *planBuilder) buildDelete(delete *ast.DeleteStmt) LogicalPlan {
 	if b.err != nil {
 		return nil
 	}
+	tableList := extractTableListFromPlan(p)
```

Comment:
p at line 1188 is not the Plan generate from `from xxx`, I need the table list, not consider where,orderby, grou